docker/php-fpm/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM php:7.2-fpm
+
+RUN apt-get update
+
+RUN apt-get install -y zlib1g-dev libpq-dev git libicu-dev libxml2-dev \
+    && docker-php-ext-configure intl \
+    && docker-php-ext-install intl \
+    && docker-php-ext-configure pgsql -with-pgsql=/usr/local/pgsql \
+    && docker-php-ext-install pdo pdo_pgsql pgsql \
+    && docker-php-ext-install zip xml
+
+RUN curl --insecure https://getcomposer.org/composer.phar -o /usr/bin/composer && chmod +x /usr/bin/composer
+
+# Set timezone
+RUN rm /etc/localtime
+RUN ln -s /usr/share/zoneinfo/Europe/Berlin /etc/localtime
```

Comment:
Why? Leave the default (UTC), as the project can be used in any timezone really!